dotCMS/src/main/java/com/dotmarketing/portlets/languagesmanager/business/LanguageCache.java

```diff
@@ -13,7 +13,9 @@
 public abstract class LanguageCache implements Cachable{
 
     protected abstract void addLanguage(Language l);
-    
+
+    protected abstract void add404Language(String languageCode, String countryCode);
```

Comment:
How did the 404 cache for a language get into this PR?

Reply:
This is explained in the description - I was because I was doing an `ab` testing to help @freddyucv reproduce an error and I saw a db hit for every page rendered